### ISSUE TITLE
Nothing happens when pressing Generate

### ISSUE THREAD

**Issue by jjazzboss** [alexgooding/auto-drummer#4]
I'm interested by your approach and wanted to test on Win10. I update the Input pattern, set the output file directory, click on Generate but nothing happens, no file is created.

**Comment by alexgooding** [comment]
A few questions.

Are you using the executable file to launch the software? 
Is nothing displaying in the UI at all when you click generate? Or does it say 'No patterns have been found.'?

**Comment by jjazzboss** [comment]
Yes, I used audodrummer.exe.   Nothing happens when I click Generate, no dialog, no error message. Is there a log file somewhere?
